tools/code_format/check_format.py

```diff
@@ -889,9 +889,11 @@ def execute_command(self, command, error_message, file_path, regex=None):
             return []
         except subprocess.CalledProcessError as e:
             if (e.returncode != 0 and e.returncode != 1):
-                return [f"ERROR: something went wrong while executing: {e.cmd}"]
+                return [
+                    f"ERROR: something went wrong while executing: {e.cmd}\n{e.output.decode()}"
```

Comment:
not sure if `utf-8` decoding is needed or not.
Leaving it up to you.

Reply:
not specifying uses the system encoding (iirc) - not 100% but i think this is supposed to be the way to do it now - ill check at some point

Reply:
chatgpt says ...

> In Python 3.11 and later versions, when using the encode and decode functions with the UTF-8 encoding, you can usually omit explicitly specifying "utf-8" as the encoding because UTF-8 is the default encoding for these functions. Python 3.11 introduced this change to improve the consistency and readability of code.

Reply:
TIL (assuming that it didn't hallucinate it :)), thanks!